### DISCUSSION TITLE
enarx user register your_username报错如下：

### DISCUSSION THREAD

**Discussion by digitalinformation** [enarx/enarx#2537] [General]
zsh@zsh-virtual-machine:~$ enarx user register zsh
Error: Failed to request device code

Caused by:
    0: Request failed
    1: ureq request failed
    2: https://auth.profian.com/oauth/device/code: Dns Failed: resolve dns name 'auth.profian.com:443': failed to lookup address information: Name does not resolve
    3: failed to lookup address information: Name does not resolve

**Comment by rjzak** [comment]
Unfortunately the `auth.profian.com` service no longer available. A replacement I ___think___ is being planned, @nickvidal would know more.

**Reply by digitalinformation** [reply]
Thank you for your help ,dear rjzak.
Can you help me please? @nickvidal
